using-social-media-as-communcation-tool/readme.md

```diff
@@ -0,0 +1,105 @@
+
+Because you only reach the people during the time of your broadcast, it can often be good to broadcast the same message multiple times and give the receiver a reason to share it every time.
+
+This is the main reason why non-profit organizations and education are so successful on social media with consumer goods, retail and e-commerce far behind in second place. Their messages are directly relatable to the people sharing them and lend a strong connotation to the sender and the sharer.
+
+## Practicality
+
+In general, people only briefly look at what they see on social media. Therefore, the first sentences or image they see should capture the reader in a way that is in line with what the message you're conveying.
+
+People hardly click on links (1%-10%), even if they have instant relevance. Therefore, you should limit use of tinyurl and bitly to urls of which the names are not relevant or obvious.
+
+Try to keep your content on a minimal amount of different sources and and channel it through social media. If using video, try to keep all your material on one provider. Try to avoid confusion where readers can find the content again.
+
+Be consistent in language, frequency, and use of external services such as Eventbrite.
+
+## Practicality per channel
+Here you will find some info specifically pertaining to the mentioned social media channels.
+
+### Twitter
+
+* Mostly used for building and maintaining a network, conveying and receiving current events to and from your followers.
+* Mixed professional and personal use.
+* Shows all the messages people you follow post in real time.
+* Mentions give a notification for the mentioned and show up in the feed of their followers.
+* If “@” is at the beginning of a sentence, the tweet is a direct message and only visible to the receiver defined by the “@”.
+* 140 character limit, images “cost” 25 characters, links 23 characters.
+* Hashtags are used for 2 things: search labels and making or taking part in a phenomenon.
+
+### Facebook
+
+* Most used and broadest channel, mostly personal use.
+* Shows messages of the people you follow based on your profile information, what and how long you view something and on what you click.
+* Suggest (sponsored) post and pages based on that information.
+* Tags give notifications to the tagged and show up in the timeline of their friends.
+* Facebook has the potential of infinite reach, so always reach beyond your followers.
+
+### LinkedIn
+
+* Used for profiling yourself and others, career-oriented.
+* Professional use.
+* Shows messages from the people you follow in real time and suggests messages, jobs, companies and connections based on your profile information and on what you click
+* 600 character limit.
+* Only the first 150 Characters are visible; the rest are hidden behind “read more”.
+
+## Conclusion
```

Comment:
Just one sentence under the 'Conclusion' header and before the bullet points would motivate me more to read the conclusion. At this point I've read so many bullet points that I don't want to keep reading.